### ISSUE TITLE
Bug in V1.0 BG corpus: English translation does not match the origin

### ISSUE THREAD

**Issue by matyaskopp** [clarin-eric/ParlaMint#12]
https://github.com/clarin-eric/ParlaMint/blob/main/ParlaMint-BG/ParlaMint-BG.xml#L422-L423

```
<event from="2014-10-27" to="2017-01-26" xml:id="NS.43">
  <label xml:lang="bg">43 Народно събрание</label>
  <label xml:lang="en">Term 7</label>
 </event>
```

It should be `Term 43`

**Comment by TomazErjavec** [comment]
Thanks for spotting this! I/we will correct these bugs in V1 when the next batch will be ready.

**Comment by TomazErjavec** [comment]
Fixed in 393e208, closing.
